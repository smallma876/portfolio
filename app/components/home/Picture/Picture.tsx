import Image from "next/image";
import React from "react";

const Picture = () => {
  return (
    <Image
      src="/images/foto.jpg"
      alt="picture-perfil"
      width={300}
      height={300}
      style={{ borderRadius: "50%" }}
    />
  );
};

export default Picture;
