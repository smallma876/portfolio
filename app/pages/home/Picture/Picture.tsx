import Image from "next/image";
import React, { FC } from "react";
import { myLoader } from "../../../../utils/loaderImages";

interface PictureProps {
  url: string;
}

const Picture: FC<PictureProps> = ({ url }) => {
  return (
    <Image
      loader={myLoader}
      src={url}
      alt="picture-perfil"
      width={300}
      height={300}
      style={{ borderRadius: "50%" }}
    />
  );
};

export default Picture;
