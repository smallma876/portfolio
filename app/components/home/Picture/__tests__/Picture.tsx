import { render } from "@testing-library/react";
import Picture from "../Picture";

const renderComponent = async () => {
  const { container } = await render(<Picture />);

  return container;
};

describe("Picture", () => {
  it("should render match snapshot", () => {
    const container = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
