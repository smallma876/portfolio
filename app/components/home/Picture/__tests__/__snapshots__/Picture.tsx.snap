// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture should render match snapshot 1`] = `Promise {}`;
